### PR TITLE
add terms dictionary

### PR DESCRIPTION
**Pull request by Lijia-Li**
- add raw terms txt obtained from http://www.musictheory.org.uk/res-musical-terms/italian-musical-terms.php
- add function prepare_terms_dict() in search to create dictionary

@justinnhli @crzeller11

### CODE REVIEW

File: search.py [hunk 1]
```diff
@@ -5,6 +5,20 @@
 
 # Generic Functions
 
+def prepare_terms_dict():
+    """
+    return a dictionary of terms for searching through abbreviation
+    Arguments: N/A
```
Comment (justinnhli) [inline search.py]: if no arguments, just leave it out of the docstring

File: search.py [hunk 2]
```diff
@@ -5,6 +5,20 @@
...
+    Return: [dictionary]: key: term name (regular spelling); value: term abbreviation
+    """
+    f = open('database/terms_dictionary.txt')
+    # sheet = [line.rstrip('\t') for line in open('terms_dictionary.txt')]
+    dic = {}
+    for line in  f:
+        line = line.split('\t')
+        if len(line) == 5:
+            dic[line[0]] = line[2].strip('()')
+    return dic
```
Comment (justinnhli) [inline search.py]: Can be turned into a comprehension:

```python
return {
    line[0]: line[2].strip('()')
    for line in line.split('\t')
    if len(line) == 5
}
```

Comment (Lijia-Li) [reply search.py]: Will do. I used `for line in open('filename')`. Let me know if that is a bad idea.

File: search.py [hunk 3]
```diff
@@ -5,6 +5,18 @@
...
+    return a dictionary of terms for searching through abbreviation
+    Return: [dictionary]: key: term name (regular spelling); value: term abbreviation
+    """
+
+    return {
+        line.split('\t')[0]: line.split('\t')[2].strip('()')
+        for line in open('database/terms_dictionary.txt')
+        if len(line) == 5
+        }
+
```
Comment (justinnhli) [inline search.py]: use a context manager instead:

```python

with open('database/terms_dictionary.txt') as file_descriptor:
    text = file_descriptor.read()
return {
    line.split('\t')[0]: line.split('\t')[2].strip('()')
    for line in text.splitlines()
    if len(line) == 5
}
```

**Pull request merged**